scripts/deploy-commands.js

```diff
@@ -17,45 +18,25 @@ If not, see <https://www.gnu.org/licenses/>.
 // You need to `tsc` before running this script.
 
 require("dotenv").config();
-
 const { REST } = require("@discordjs/rest");
 const { Routes } = require("discord.js");
+const CommandImportService = require("../dist/services/CommandImportService").default;
 
-const { getOrLoadAllCommands } = require("../dist/handlers/commandHandler");
-const { getApiToken, getGuildId, getClientId } = require("../dist/helpers/configHelpers");
-
-const API_TOKEN = getApiToken();
-const CLIENT_ID = getClientId();
-const GUILD_ID = getGuildId();
-
-const isGlobal = process.argv.some(x => x === "--global");
+const { DISCORD_API_TOKEN, CLIENT_ID } = process.env;
 const isUndeploy = process.argv.some(x => x === "--undeploy");
 
-if (!API_TOKEN || !CLIENT_ID) {
+if (!DISCORD_API_TOKEN || !CLIENT_ID) {
```

Comment:
I don't even think you can set the environment variables to an empty string, but good point, I'll see if I can do something easy about it, otherwise I'll just close

Reply:
God save us if you can.